web/pano_editor.css

```
@@ -551,368 +602,368 @@
+.pano-paint-color-dot.active {
+  filter: saturate(1.06);
+}
+.pano-paint-color-dot.active::before {
+  inset: 4px;
+}
+.pano-paint-color-dot.active::after {
+  box-shadow: inset 0 0 0 2.5px var(--swatch, #fff);
+  opacity: 1;
+}
+.pano-paint-color-dot-rainbow {
+}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)